.travis.yml

```diff
@@ -1,7 +1,47 @@
 language: php
+
+sudo: false
+
+cache:
+    directories:
+        - $HOME/.composer/cache
+
 php:
-    - 5.5
     - 5.6
     - 7.0
-install: composer install --no-interaction
-script: ./vendor/bin/phpunit
+    - 7.1
+
+env:
+    global:
+        - MOODLE_BRANCH=MOODLE_32_STABLE
+    matrix:
+        - DB=pgsql
+        - DB=mysqli
+
+before_install:
+    - export ORIG="$(pwd)"
+    - phpenv config-rm xdebug.ini
+    - nvm install node
+    - cd ../..
+    - composer selfupdate
+    - composer create-project -n --no-dev --prefer-dist moodlerooms/moodle-plugin-ci ci ^2
+    - export PATH="$(cd ci/bin; pwd):$(cd ci/vendor/bin; pwd):$PATH"
+
+install:
+    - moodle-plugin-ci install
+
+script:
+#   - moodle-plugin-ci phplint
+#   - moodle-plugin-ci phpcpd
+#   - moodle-plugin-ci phpmd
+    - moodle-plugin-ci codechecker
```

Comment:
As far as `moodle-plugin-ci codechecker` goes, it's possible to run `phpcs` locally against the Moodle coding style by following [these instructions](https://github.com/moodlerooms/moodle-coding-standard).

It's apparently possible to [run a Travis build locally](https://stackoverflow.com/questions/21053657/how-to-run-travis-ci-locally) so that's possibly a way in which to run the other `moodle-plugin-ci` items.